exonum/src/api/node/public/explorer.rs

```diff
@@ -211,17 +278,55 @@ impl ExplorerApi {
         );
         api_scope
             .endpoint("v1/blocks", Self::blocks)
+            .endpoint("v1/blocks_with_time", Self::blocks_with_time)
```

Comment:
`BlocksRange` structure can be extended to holding optional times:
```rust
pub struct BlocksRange {
    pub range: Range<Height>,
    pub blocks: Vec<Block>,
    pub times: Vec<DateTime<Utc>>,
}
```
But I don't like this approach because these vectors don't necessary have the same size. In the current approach the type system guarantees it.

Still, it wouldn't be a breaking change for external api users.